mlops/common/pipeline_utils.py

```diff
@@ -0,0 +1,172 @@
+"""
+This module defines a machine learning pipeline for processing, training, and evaluating data.
+"""
+from azure.identity import DefaultAzureCredential
+from azure.ai.ml.dsl import pipeline
+from azure.ai.ml import MLClient
+import time
+
+from mlops.common.config_utils import MLOpsConfig
+from mlops.common.get_compute import get_compute
+from mlops.common.naming_utils import generate_experiment_name, generate_run_name
+
+
+def set_pipeline_properties(
```

Comment:
i'd consider making this more dynamic using a python dict datatype. 

this way any pipeline can specify its own key:value tags, and the `pipeline_job.tags` value can simply be that dict param

Reply:
updated this!